components/card/card.scss

```diff
@@ -0,0 +1,48 @@
+.ui-card {
+  border-radius: var(--tx-generic-border-radius);
+  flex: 1 1 auto;
+  position: relative;
+  background: var(--tx-generic-color-blank);
+  border: var(--tx-generic-border-width) var(--tx-generic-border-style) var(--tx-generic-color-blank);
+}
+
+.ui-card_checked.ui-card_show-icon {
+  &:after {
```

Comment:
please avoid using SCSS features, you can have here simple CSS file which will not be transpiled

Reply:
but why? we have scss in ui-kit, are we going to migrate from SCSS to CSS or css modules?

Reply:
probably. I'm just saying that you're not using here anything special that could be achieved only with SCSS. Keeping it simple will make any migration in future easier, IMO